### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by sujfuy** [mifi/lossless-cut#2145] [Q&A]
Not sure how I did this the first time, but I was wanting to update to the latest version but I cannot extract the files to a program folder...  I currently have a shortcut icon in my start menu, but not sure how I extracted the files to Programs. Windows 11 64 bit.

**Comment by hrxn** [comment]
Download this
https://github.com/mifi/lossless-cut/releases/download/v3.62.0/LosslessCut-win-x64.7z

Use 7zip to extract:
https://github.com/ip7z/7zip

**Reply by sujfuy** [reply]
I've got the zip file, but windows will not let me extract it to the folder I created in Programs.  I already deleted the old one.

**Reply by sujfuy** [reply]
I get this error code:  error 0x80070005: Access is Denied when I attempt to extract to a folder I created in Programs X86 for LosslessCut

**Reply by hrxn** [reply]
Well, if it's an access rights issue with Windows, either change permissions for your Program directory, or simply use another directory where you have full write access?

**Comment by sujfuy** [comment]
I finally got 'er done....  had to battle windows as usual.  Not sure why it was a problem this time and wasn't the last time.  Loving this program.  Thanks
